SR830/SR830.py

```diff
@@ -44,9 +44,9 @@ def getTC(i):
     elif i > 19:
         return getTC(19)
     elif i % 2 == 0:
-        return 10**(-5 + i/2) * units.s
+        return 10**(-5 + i/2) * s
```

Comment:
I don't think `s` is the unit object you want, and also it's a fairly generic name that gets used for other stuff. I'd suggest leaving this as `units.s`, or else import `Unit` and use `Unit('s')`